webapp/components/test-runs-query.html

```diff
@@ -72,11 +73,15 @@
           this.productSpecs = (products || []).map(p => this.getSpec(p));
         }
 
-        computeTestRunQueryParams(sha, aligned, labels, productSpecs, maxCount, from, to) {
+        computeTestRunQueryParams(sha, aligned, master, labels, productSpecs, maxCount, from, to) {
           const params = {};
           if (!this.computeIsLatest(sha)) {
             params.sha = sha;
           }
+          if (this.masterRunsOnly && master
+              && (!labels || !labels.includes('master'))) {
+            labels = (labels || []).concat('master');
```

Comment:
This should be `labels = (labels || []).concat(['master']);` or `labels = labels || []; labels.push('master');`

Reply:
We don't want to mutate the UI property (potentially there was a bug before this PR). Your code is more readable though; changed.